code/obj/submachine/chem_extractor.dm

```diff
@@ -22,8 +22,9 @@ TYPEINFO(/obj/submachine/chem_extractor)
 	var/obj/item/reagent_containers/glass/storage_tank_2 = null
 	var/list/ingredients = list()
 	var/list/allowed = list(/obj/item/reagent_containers/food/fish/, /obj/item/reagent_containers/food/snacks/,/obj/item/plant/,/obj/item/clothing/head/flower/,/obj/item/seashell)
+	var/obj/item/robot_chemaster/prototype/parent_item = null
 
-	New()
+	New(var/obj/item/robot_chemaster/prototype/parent_item = null)
```

Comment:
Isn't the first argument of `New` the location of where it's being instantiated? Does this still work if instantiating on a turf?

Reply:
Turns out setting location to null breaks the child machines and results in "[src] seems to be out of order." messages. Setting loc to the parent item like before seems to work well, and looking at `/obj/machinery/proc/powered(...)` makes me think obj locs are fine